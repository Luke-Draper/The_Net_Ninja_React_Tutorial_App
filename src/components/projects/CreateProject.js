import React, { Component } from "react";

class CreateProject extends Component {
	render() {
		return (
			<div className="CreateProject">
				<p>CreateProject</p>
			</div>
		);
	}
}

export default CreateProject;
