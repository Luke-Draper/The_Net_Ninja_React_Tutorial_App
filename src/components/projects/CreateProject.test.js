import React from "react";
import CreateProject from "./CreateProject";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("CreateProject renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<CreateProject />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
