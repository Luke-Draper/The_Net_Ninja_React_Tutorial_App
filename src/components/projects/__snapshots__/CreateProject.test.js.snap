// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateProject renders without crashing 1`] = `
<div
  className="create-project"
>
  <h2>
    CreateProject
  </h2>
</div>
`;
